db/schema.rb

```diff
@@ -494,23 +370,23 @@
     t.boolean "stem_indicator", default: false
     t.string "campus_type"
     t.string "parent_facility_code_id"
-    t.index "lower((institution)::text) text_pattern_ops", name: "institutions_temp_lower_idx"
-    t.index ["address_1"], name: "institutions_temp_address_1_idx"
-    t.index ["address_2"], name: "institutions_temp_address_2_idx"
-    t.index ["address_3"], name: "institutions_temp_address_3_idx"
-    t.index ["city"], name: "institutions_temp_city_idx"
-    t.index ["cross"], name: "institutions_temp_cross_idx"
-    t.index ["distance_learning"], name: "institutions_temp_distance_learning_idx"
-    t.index ["facility_code"], name: "institutions_temp_facility_code_idx"
-    t.index ["institution"], name: "institutions_temp_institution_idx"
-    t.index ["institution_type_name"], name: "institutions_temp_institution_type_name_idx"
-    t.index ["online_only"], name: "institutions_temp_online_only_idx"
-    t.index ["ope"], name: "institutions_temp_ope_idx"
-    t.index ["ope6"], name: "institutions_temp_ope6_idx"
-    t.index ["state"], name: "institutions_temp_state_idx"
-    t.index ["stem_offered"], name: "institutions_temp_stem_offered_idx"
-    t.index ["version", "parent_facility_code_id"], name: "institutions_temp_version_parent_facility_code_id_idx"
-    t.index ["version"], name: "institutions_temp_version_idx"
```

Comment:
Remove these index renamings please.  We'll handle those in a separate PR.

Reply:
This happens when running `bundle exec rake db:migrate`

Reply:
removed